librarian-puppet.gemspec

```diff
@@ -25,17 +25,17 @@ Gem::Specification.new do |s|
 
   s.executables = ['librarian-puppet']
 
-  s.add_dependency "librarianp", ">=0.6.3"
-  s.add_dependency "rsync"
+  s.add_dependency "librarianp", '~> 1.1'
+  s.add_dependency "rsync", '~> 1.0'
   s.add_dependency "puppet_forge", ">= 2", '< 7'
 
-  s.add_development_dependency "rake"
-  s.add_development_dependency "rspec"
+  s.add_development_dependency "rake" , '~> 13.2'
+  s.add_development_dependency "rspec", '~> 3.13'
   s.add_development_dependency "cucumber", '~> 9.2'
   s.add_development_dependency "aruba", ">= 1.0", "< 3"
-  s.add_development_dependency "puppet", ENV["PUPPET_VERSION"]
+  s.add_development_dependency "puppet", '>= 7', '< 9'
```

Comment:
I think `Gemfile` overrides this. It may be better to just define all dev dependencies in `Gemfile`.

Reply:
I think the majority of our gems has their dev deps in the gemspec. The benefit is that they are properly tracked on rubygems. But I don't have strong preference. If you want it can move all of them to the Gemfile.

Reply:
https://docs.rubocop.org/rubocop/cops_gemspec.html#gemspecdevelopmentdependencies isn't enabled by default, but it has an opinion.